users/api/urls.py

```diff
@@ -3,6 +3,6 @@
 from .views import ProfileDetailAPIView, ProfileUpdateAPIView
 
 urlpatterns = [
-    path('profile/<user>/', ProfileDetailAPIView.as_view(), name='detail'),
-    path('profile/<user>/update/', ProfileUpdateAPIView.as_view(), name='update'),
+    path('profile/', ProfileDetailAPIView.as_view(), name='detail'),
```

Comment:
add <user_id> support to this and create a new `/myprofile` route

Reply:
Completed